designer/server/src/routes/forms/edit.test.js

```diff
@@ -0,0 +1,106 @@
+import { StatusCodes } from 'http-status-codes'
+
+import { createServer } from '~/src/createServer.js'
+import * as forms from '~/src/lib/forms.js'
+import { auth } from '~/test/fixtures/auth.js'
+import {
+  renderResponse,
+  serverResponse
+} from '~/test/helpers/component-helpers.js'
+
+jest.mock('~/src/lib/forms.js')
+
+describe('Forms library routes', () => {
+  /** @type {import('@hapi/hapi').Server} */
+  let server
+
+  beforeAll(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  const now = new Date()
+  const authorId = 'f50ceeed-b7a4-47cf-a498-094efc99f8bc'
+  const authorDisplayName = 'Enrique Chase'
+
+  /**
+   * @satisfies {FormMetadataAuthor}
+   */
+  const author = {
+    id: authorId,
+    displayName: authorDisplayName
+  }
+
+  /**
+   * @satisfies {FormMetadata}
+   */
+  const formMetadata = {
+    id: '661e4ca5039739ef2902b214',
+    slug: 'my-form-slug',
+    title: 'Test form',
+    organisation: 'Defra',
+    teamName: 'Defra Forms',
+    teamEmail: 'defraforms@defra.gov.uk',
+    createdAt: now,
+    createdBy: author,
+    updatedAt: now,
+    updatedBy: author
+  }
+
+  /**
+   * @satisfies {FormDefinition}
+   */
+  const formDefinition = {
+    name: 'Test form',
+    pages: [],
+    conditions: [],
+    sections: [],
+    lists: []
+  }
+
+  test('GET - should check correct formData is rendered in the view', async () => {
+    jest.mocked(forms.get).mockResolvedValueOnce(formMetadata)
+    jest
+      .mocked(forms.getDraftFormDefinition)
+      .mockResolvedValueOnce(formDefinition)
+
+    const options = {
+      method: 'get',
+      url: '/library/my-form-slug/edit/team',
+      auth
+    }
+
+    const { document } = await renderResponse(server, options)
+
+    const teamName = document.querySelector('#teamName')
+    const teamEmail = document.querySelector('#teamEmail')
+
+    expect(teamName?.value).toBe('Defra Forms')
+    expect(teamEmail?.value).toBe('defraforms@defra.gov.uk')
```

Comment:
tsc checks:

```
designer/server/src/routes/forms/edit.test.js(78,22): error TS2339: Property 'value' does not exist on type 'Element'.
designer/server/src/routes/forms/edit.test.js(79,23): error TS2339: Property 'value' does not exist on type 'Element'.
```

Reply:
I don't see this error now but there are errors not related to PR. Could you please check?